docs/howto/testing.md

```diff
@@ -18,6 +18,16 @@ To write a test, place a Javascript file with the `-test.js` suffix in the
 `__tests__` directory inside of any subfolder of `/src`. The test will be
 automatically picked up by the test runner.
 
+We've configured Jest to run two different
+["projects"](https://jestjs.io/docs/configuration#projects-arraystring--projectconfig):
+one with an iOS preset, and one with an Android preset.
+`tools/test jest` will run one or the other, or both, depending on
+what you pass for `--platform`.
+
+If it makes sense for a test file to be dedicated to just one
+platform, and that's unlikely to change, you can instead suffix it
+with `-test.ios.js` or `-test.android.js`, and it will be ignored when
+the other platform's project runs.
```

Comment:
This feels more detailed about the Jest specifics than is needed here -- those details can be left to the code that controls them. (E.g. I think the concept of a Jest "project" can be left to `jest.config.js` and to that `case` block in `tools/test` that sets `--selectedProjects`.)

For this doc the ideal thing is to be more direct about what happens by default, and what you might want to do instead (and when.)